### ISSUE TITLE
Support biorhythm additionally to astro functions.

### ISSUE THREAD

**Issue by renne** [alisdairjsmyth/node-red-contrib-blindcontroller#32]
Currently blinds are closed between sunset and sunrise (e.g. between 22:30 and 03:30).
It would be great to either set times (e.g. 22:00 to 06:00) or a bool variable to close blinds according to human biorhythm.

**Comment by alisdairjsmyth** [comment]
@renne - what would be the relationship between human biorhythm and blind position?

I like the idea of including set times as an alternative to sunrise and sunset.

**Comment by renne** [comment]
Personally I use a variable to decide between active (e.g. bright + warm) and passive (dark + cold) biorhythm phase. In my case active phase is from 06:00 AM to 09:00 PM. You can also add a third state "relax" (dimmed light, slightly colder).

**Comment by TroLoos** [comment]
Guys, any chance we could revive this idea?

I love the idea to steer my blinds according to sunlight and sun position. But my morning and night modes are as much important...

I have a small kid and a blind opening at 4 am in the morning wakes him up and brings me trouble. I think there should be a way to set boundary times manually. How about it?

**Comment by MRobi1** [comment]
I would agree with @TroLoos here. I use modes (Morning, Afternoon, Evening, Night/Sleep) to control various things in the house. Night/Sleep mode is on when pressure sensors detect we're in bed. I also don't need blinds opening at sunrise at 5am when the alarm is set for 7:30. The ability to keep them closed until Morning, triggered by getting out of bed, would be great!
